regression/verilog/expressions/shl4.desc

```diff
@@ -0,0 +1,9 @@
+KNOWNBUG
```

Comment:
That's a bit scary that we are giving the wrong answer for such a basic test?

Reply:
Yes; it's scary semantics.